Gems/Atom/RPI/Code/Source/RPI.Public/RPISystem.cpp

```diff
@@ -119,10 +119,6 @@ namespace AZ
             Debug::TraceMessageBus::Handler::BusConnect();
 #endif
             m_descriptor = rpiSystemDescriptor;
-
-            // set the default multisample state to MSAA 4x
-            // the default render pipeline may override this when it is loaded
-            m_multisampleState.m_samples = 4;
```

Comment:
did you mean to remove this?

Reply:
It is kinda unrelated, but it also seems quite wrong. I think it was set to 4 back when we used to use msaa 4x as the norm, but we use 2x more regularly now, and the RPI shouldn't assume msaa usage at all.